code/datums/setup_option/backgrounds/origin_upbringing.dm

```diff
@@ -111,15 +111,15 @@
 	desc = "Your entire life has been a series of unlucky and often self-inflicted accidents, you spent enough time hurting yourself due to your own clumsiness that you've built up a more \
 	pain tolerance than most of common folks, meaning that due to this clumsiness problem, you've become quite tough and a little stronger too. A shame, this doesn't stop you from time to time failing even the most basic tasks at times. In fact, you find some tasks that require precision damn near impossible, and even handling or using guns is a dangerous prospect. Due to this problem, you have a terrible perception around the area around you."
 
-	perks = list(/datum/perk/klutz)
+	perks = list(/datum/perk/klutz, /datum/perk/clumsy)
 
 	stat_modifiers = list(
```

Comment:
What in the actual hell are these modifier numbers. This is super fucking up there. Like, specialist synthetic levels of up there. That's kinda fucking insane and I don't personally see this flying.

Reply:
Probably should've commented this lower in the file tree.

Reply:
I am very very bad at balancing things LOL. Just give me a suggested number and i will apply it

Reply:
> Probably should've commented this lower in the file tree.

I still want to buff the stats but without being really overboard.
What can i do so it is not that bad?

Reply:
The numbers it's at normally are already a very very strong roundstart buff, I'd just suggest not changing them at all, personally.

Reply:
The rest of the changes I can't really speak on but those numbers are, whewf.

Reply:
To give a bit of understanding, 45 TGH is something akin to what the Warrant Officer or Blackshield Commander spawn with, somewhere around there without me actually going to check the file to give that with evidence. Klutz gave a lot of points compared to everything else pickable in background because it heavily debuffed you and made you melee-only essentially.

Reply:
then i will lower the number a bit. And yes i suggest changing to take the account of the vigilance debuff that is to the account of making clumsy mutation less awful but that is not up to me i suppose

Reply:
Well its not quite up to me either, I'm just saying that like. This is kinda INANELY good stats for LESSENING the impact of a debuff to get the stats.

Reply:
> To give a bit of understanding, 45 TGH is something akin to what the Warrant Officer or Blackshield Commander spawn with, somewhere around there without me actually going to check the file to give that with evidence. Klutz gave a lot of points compared to everything else pickable in background because it heavily debuffed you and made you melee-only essentially.

Oh i understand.
I just wished to give them due to how i made the changes and the chances. If anything elses comes in the matter and making the whole stunning yourself and shooting yourself with percentage bigger. It would probably be nice to have somehow more stats to compensate over it aside that?

Reply:
> Well its not quite up to me either, I'm just saying that like. This is kinda INANELY good stats for LESSENING the impact of a debuff to get the stats.

I still have in mind in adding more interactions for klutz in a future PR.
Like if bandaging yourself you have chance of smashing the bandage on your wound hurting more, and alot more interactions but i still want to make it playlabe before that